### ISSUE TITLE
Exposure to propofol infusion

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10268]
**Preferred term label:**
Exposure to propofol infusion

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://uts.nlm.nih.gov/uts/umls/concept/C0567883

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Propofol infusion syndrome

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
It seems this is related to long-term sedation with propofol (https://associationofanaesthetists-publications.onlinelibrary.wiley.com/doi/10.1111/j.1365-2044.2007.05055.x), and since propofol is used so often (single use) for anesthesia, I wonder if it would be better to modify the label to something like
Recent history of sedation by propofol infusion

**Comment by MickeySegal** [comment]
That is fine, though we had a case in Massachusetts of a kid having this after one dose.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000883
- label: Recent history of sedation by propofol infusion
- parents: [Recent past medical history (HP:4000144)](https://hpo.jax.org/app/browse/term/HP:4000144)
- pmid: PMID:17567345
- def: A history of exposure to anesthesia by propofanol infusion recently before the presenting complaint.
- comment: Propofol is commonly used in anaesthesia and in intensive care where its favourable pharmacokinetics and rapidly reversible sedation make it an ideal drug for short-term sedation. However, propofol has been associated with a number of serious adverse effects such as metabolic acidosis, cardiac asystole, myocardial failure, rhabdomyolysis, and death.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10268
- synonyms:
